README.md

```diff
@@ -191,6 +189,18 @@ by the installation.
 Again, it is advised to verify that Perun is running correctly in your environment as follows:
 
     make test
+
+Alternatively, you can use [Tox](https://tox.wiki/en/latest/) to run tests for all the supported
+Python versions, as well as run static type checking, code style linting and generating
+documentation:
+
+    tox run
+
+If you wish to test only against a single Python version, run Tox as:
+
+    tox run -e pyXY
+
+where `XY` is the python version you wish to test. 
```

Comment:
Suggestion: write this as `py3.X` maybe? and mention which version we currently safely support?

Reply:
The `pyXY` is actually a naming convention used by Tox to automatically deduce the Python interpreter version (see https://tox.wiki/en/latest/config.html#basepython), but it seems that it understands `pyX.Y` as well (no mention of this in docs tho?), so I updated it. As for the supported versions, I have updated the README with the command to show available Tox environments.

Reply:
OK, that's fine then.